### PR TITLE
User guide grammar fixes

### PR DESCRIPTION
**Pull request by lxgt43**
# Summary

This PR adds a few grammar and formatting changes to the user guide. I tried to make it more readable, make some terms consistent and remove some ambiguity. The PR is likely opinionated in some places, I am open to reverting changes as needed.

# Changes

- Changed the Markdown examples table to display code blocks and lists using HTML rather than plain text.
- Changed wording in a few places.
- Made the spelling of a few terms more consistent: Markdown - capitalized it everywhere; front page - two words everywhere.
- Added sectioning to large blocks of text.
- Capitalized headers and changed all top-level headers to H1 (and nested ones H2, H3 etc).

### CODE REVIEW

File: src/users/04-moderation.md [hunk 1]
```diff
@@ -1,33 +1,35 @@
...
+The internet is full of bots, trolls and other malicious actors. Sooner or later they will post unwanted content to any website that is open to the public. It is the task of administrators and moderators to remove such unwanted content. Lemmy provides many tools for this, from removing individual posts and issuing temporary bans, to removing all content submitted by an offending user.
 
-Moderation in Lemmy is divided between administrators and moderators. Admins are responsible for the entire instance, and can take action on any content. They are also the only ones who can completely ban users. In contrast, moderators are only responsible for a single community. Where admins can ban a user from the entire instance, mods can only ban them from their community.
+Moderation in Lemmy is divided between administrators (admins) and moderators (mods). Admins are responsible for the entire instance, and can take action on any content. They are also the only ones who can completely ban users. In contrast, moderators are only responsible for a single community. Whereas admins can ban a user from the entire instance, mods can only ban them from their community.
```
Comment (dessalines) [inline src/users/04-moderation.md:5]: Could you use a markdown italics or bold for *admins* and *mods*

**Review by dessalines — COMMENTED** [review]
Just one recommendation, all the other changes are great. Thx for this!

File: src/users/04-moderation.md (hunk 1 above)
Comment (lxgt43) [reply src/users/04-moderation.md]: Good point, thanks! Added italics

**Review by dessalines — APPROVED** [review]
Sweet, thx!

**Pull request merged**